res/css/views/beta/_BetaCard.scss

```diff
@@ -119,7 +119,7 @@ limitations under the License.
     font-size: 12px;
     font-weight: $font-semi-bold;
     line-height: 15px;
-    color: #FFFFFF;
+    color: $background;
```

Comment:
This makes beta pills have the wrong text color in dark mode:

![Screenshot 2022-04-24 at 13-58-10 Element Test room](https://user-images.githubusercontent.com/48614497/164989970-b06b987b-0800-4af3-bde7-c2b01c0e63e0.png)

I think `$button-primary-fg-color` is the correct variable here.